### ISSUE TITLE
"Failed to load map!...TestCapturer.umap appears to be an asset file." error when opening NDDS

### ISSUE THREAD

**Issue by trevoravant** [NVIDIA/Dataset_Synthesizer#43]
When trying to load NDDS into Unreal Engine I'm getting the error
  
> Failed to load map!
> ../../../../Dataset_Synthesizer/Source/Content/Map/TestCapturer.umap
> appears to be an asset file.

---

**steps I followed:**

I installed Unreal Engine on Arch Linux, following the instructions on [this page](https://wiki.unrealengine.com/Building_On_Linux). Note that I downloaded the Unreal Engine source code using the command
`git clone -b 4.22 https://github.com/EpicGames/UnrealEngine.git`
which installed version 4.22.3 on my system.

I then download NDDS using the command

`git lfs clone https://github.com/NVIDIA/Dataset_Synthesizer.git`

which resulted in a directory of size 406MB.

I then ran Unreal Engine:
```
cd UnrealEngine
./Engine/Binaries/Linux/UE4Editor
```

In the _Unreal Project Browser_ dialog box, I clicked _Browse..._, selected _DatasetSynthesizer/Source/NDDS.uproject_ and clicked _Open_. I got the _Convert Project_ dialog box so (as the NDDS documentation suggests) I chose _More Options..._ then _Skip conversion_. This generated the _Missing NDDS Modules_ dialog box asking if I'd like to rebuild some modules, so (as the NDDS documentation suggests) I chose _Yes_.

I then got the error message
![error_message](https://user-images.githubusercontent.com/45779884/69840552-dc1ebc80-1210-11ea-8012-6f1c18a1f860.png)

If click *OK*, Unreal Editor still opens, but NDDS obviously didn't load correctly

---

I don't have too much experience with Unreal Engine, so I'm not sure what this means or what to do. Any help would be appreciated.

**Comment by smitheric95** [comment]
I've gotten this error in the past but just ignored it. I believe this means that Unreal just failed to open one level in particular, but the NDDS plugin was probably loaded successfully. Perhaps the file got corrupted?

Are you able to create your own levels and export data as the documentation describes? Page 10 of the documentation.

**Comment by trevoravant** [comment]
Ah, figured it out! The problem was due to how I git cloned the Dataset_Synthesizer repository. I had never initialized git lfs using the command
```
git lfs install
```
Once I did that, I was able to clone the repository using either the original `git lfs clone` command I posted, or just
```
git clone https://github.com/NVIDIA/Dataset_Synthesizer.git
```
I ended up with directory of size 788 MB. With my original approach, I only had a directory of size 406 MB, so I probably missed some files.
